snippets/nodejs/getHostsList.js

```diff
@@ -0,0 +1,38 @@
+var netbackup = require("./netbackup-module/netbackup.js");
+// including stdio just for creating help
+var stdio = require('stdio');
+var parms = stdio.getopt({
+    'nbmaster': { key: 'n', args: 1, description: 'Master server name', mandatory: true },
+    'port'    : { key: 'pr', args: 1, default: '1556', description: 'Port number' },
+    'username': { key: 'u', args: 1, description: 'User name', mandatory: true },
+    'password': { key: 'p', args: 1, description: 'Password of a user', mandatory: true },
+    'domain'  : { key: 'd', args: 1, default: '', description: 'Domain name (empty by default)' },
+    'verbose' : { key: 'v', args: 1, description: 'Verbose statements' }
+});
+
+var contentType = "application/vnd.netbackup+json;version=2.0";
```

Comment:
If the script is for 8.1.1 as well, then the API version should be kept at 1.0. 8.1.1 server won't be able to serve 2.0.